### ISSUE TITLE
INCOMPATIBILITY WITH FUTURE HA Core 2025.1

### ISSUE THREAD

**Issue by robitsrl** [krahabb/meross_lan#362]
I'm using Meross LAN integraion in local mode (no cloud) with psx340 Smart Surge Protector

I found this in the log:

ELECTRIC_CURRENT_AMPERE was used from meross_lan, this is a deprecated constant which will be removed in HA Core 2025.1. Use UnitOfElectricCurrent.AMPERE instead, please create a bug report at https://github.com/krahabb/meross_lan/issues
ELECTRIC_POTENTIAL_VOLT was used from meross_lan, this is a deprecated constant which will be removed in HA Core 2025.1. Use UnitOfElectricPotential.VOLT instead, please create a bug report at https://github.com/krahabb/meross_lan/issues
ENERGY_WATT_HOUR was used from meross_lan, this is a deprecated constant which will be removed in HA Core 2025.1. Use UnitOfEnergy.WATT_HOUR instead, please create a bug report at https://github.com/krahabb/meross_lan/issues
POWER_WATT was used from meross_lan, this is a deprecated constant which will be removed in HA Core 2025.1. Use UnitOfPower.WATT instead, please create a bug report at https://github.com/krahabb/meross_lan/issues
TEMP_CELSIUS was used from meross_lan, this is a deprecated constant which will be removed in HA Core 2025.1. Use UnitOfTemperature.CELSIUS instead, please create a bug report at https://github.com/krahabb/meross_lan/issues

**Comment by jaal2001** [comment]
You can also add this one 
WARNING (MainThread) [homeassistant.const] TIME_SECONDS was used from meross_lan, this is a deprecated constant which will be removed in HA Core 2025.1. Use UnitOfTime.SECONDS instead, please create a bug report at https://github.com/krahabb/meross_lan/issues

**Comment by jamespreedy** [comment]
+1 here, thanks in advance for your help!

```
Logger: homeassistant.const
Source: helpers/deprecation.py:204
First occurred: 7:57:45 am (26 occurrences)
Last logged: 7:57:48 am

TIME_SECONDS was used from meross_lan, this is a deprecated constant which will be removed in HA Core 2025.1. Use UnitOfTime.SECONDS instead, please create a bug report at https://github.com/krahabb/meross_lan/issues
```

**Comment by Joo01** [comment]
+1 , at using of "msg100" Garage Door Opener

**Comment by krahabb** [comment]
Yeah...nice it is due in 2025.1 ;)
The first bunch of deprecations were fixed in 4.5.1 (which is already live) thanks to #354 .
But the TIME_SECONDS went unnoticed so it will come in the next update(s) plenty of time!
